interface/src/stake_history.rs

```diff
@@ -65,6 +65,8 @@ impl std::ops::Add for StakeHistoryEntry {
 #[derive(Debug, PartialEq, Eq, Default, Clone)]
 pub struct StakeHistory(Vec<(Epoch, StakeHistoryEntry)>);
 
+declare_sysvar_id!("SysvarStakeHistory1111111111111111111111111", StakeHistory);
```

Comment:
Elsewhere we're re-exporting whatever's declared in solana-sdk-ids, but since this is outside the SDK, I think it makes more sense to redefine it.

Reply:
Oh I would have preferred to just re-export from solana-sdk-ids but it's not published yet

Reply:
I've been doing some prereleases of all the new crates, so https://crates.io/crates/solana-sdk-ids/0.0.3 probably has what you want :eyes: